<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formation</class>
 <widget class="QMainWindow" name="Formation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
font:  10pt  &quot;Calibrii&quot; ;
font-weight: bold
 }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Date de la formation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nom de la formation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_nom_formation">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Type de formation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_type_formation">
          <item>
           <property name="text">
            <string>Continue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Initiale</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Domaine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_domaine"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Plan de la formation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_plan_formation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Tableau resultats</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>NOM</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>PRENOM</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>RESULTAT</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>NOTE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>REMARQUE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>PRESENCE</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_enregistrer">
        <property name="text">
         <string>Enregistrer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_annuler">
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
